### DISCUSSION TITLE
GMTSAR does not give unwrap files

### DISCUSSION THREAD

**Discussion by MohammadAminSh1996** [gmtsar/gmtsar#695] [General]
Hello dear colleague
I use GMTSAR in Ubuntu VirtualBox and my files includes: (for example)
S-1-20200315.SAFE
S-1-20220315.SAFE
S-1-20200315.EOF
S-1-20220315.EOF
DEM(topo) files
is
And I also run the following codes:
./run_preproc_align.sh <SLC1.SAFE filename> <SLC2.SAFE filename> 
<POEORB1 EOF filename> <POEORB2 EOF filename>

./run_tops.sh
After a few hours and doing various processes in the terminal, all the files related to the replacement, phase, and… ready for me and are executable. But unwrap file does not work for me and cannot be used. Is there a reason for this ?
Thank you for your guidance.

**Comment by Xiaohua-Eric-Xu** [comment]
I think these scripts are not from the GMTSAR software, so I am not sure what happened. You probably need to contact the author of those scripts in order to figure out what's the problem.

./run_preproc_align.sh <SLC1.SAFE filename> <SLC2.SAFE filename>
./run_tops.sh

**Reply by MohammadAminSh1996** [reply]
![jjjn](https://user-images.githubusercontent.com/128274158/227768511-a93f0b92-439a-41f8-8cb8-10db3553a6ea.PNG)
Thanks for your guidance. 
But I downloaded GMTSAR from your website and I run the codes in the terminal. I excute my codes from the this PDF (page 56 and 61):
https://www.researchgate.net/publication/354780485_GMTSAR_installation_and_processing_Guide_-_Practical_InSAR_handbook_series


this codes generate all files except unwrap files.

**Reply by Xiaohua-Eric-Xu** [reply]
I know you did that, but you'll need to contact the author of that script since we don't know what someone's script does. Or just run `snaphu.csh 0.1 0` over that directory and send us the log file.
